src/components/vm/disks/diskEdit.tsx

```diff
@@ -55,12 +75,24 @@ const NameRow = ({ idPrefix, name, diskType }) => {
     );
 };
 
-const CacheRow = ({ onValueChanged, dialogValues, idPrefix, shutoff }) => {
+const CacheRow = ({
+    onValueChanged,
+    dialogValues,
+    idPrefix,
+    shutoff
+} : {
+    onValueChanged: OnValueChanged,
+    dialogValues: DialogValues,
+    idPrefix: string,
+    shutoff: boolean,
+}) => {
     return (
         <FormGroup fieldId={`${idPrefix}-cache-mode`}
                    label={_("Cache")}
-                   labelHelp={!shutoff &&
-                       <InfoPopover bodyContent={_("Machine must be shut off before changing cache mode")} />}>
+                   {...!shutoff
+                       ? { labelHelp: <InfoPopover bodyContent={_("Machine must be shut off before changing cache mode")} /> }
+                       : { }
+                   }>
```

Comment:
I figure that'd be a good reason to declare `labelHelp` as `| undefined` instead of `?` ? (Don't worry, it's fine, it's just inconvenient)